editor/project_settings_editor.cpp

```diff
@@ -39,6 +39,7 @@
 #include "editor/editor_undo_redo_manager.h"
 #include "editor/export/editor_export.h"
 #include "scene/gui/check_button.h"
+#include "scene/gui/separator.h"
```

Comment:
What is this for?

Reply:
A remnant from an experiment that I forgot to take out, fixed